docs/src/Box.doc.js

```diff
@@ -480,143 +540,235 @@ card(
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      description={`Borders are controlled by the \`borderStyle\` property. Specifying a size ("sm" or "lg") enables a solid, light gray color in that width, while specifying "shadow" adds a box-shadow instead.`}
+      title="Borders"
+    >
+      <MainSection.Card
+        cardSize="sm"
+        title='borderStyle="sm"'
+        defaultCode={`<Box
+        width={60}
+        height={60}
+        rounding="circle"
+        color="white"
+        borderStyle="sm"
+      />`}
+      />
+      <MainSection.Card
+        cardSize="sm"
+        title='borderStyle="lg"'
+        defaultCode={`<Box
+        width={60}
+        height={60}
+        rounding="circle"
+        color="white"
+        borderStyle="lg"
+      />`}
+      />
+      <MainSection.Card
+        cardSize="sm"
+        title='borderStyle="shadow"'
+        defaultCode={`<Box
+        width={60}
+        height={60}
+        rounding="circle"
+        color="white"
+        borderStyle="shadow"
+      />`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection description="" title="Colors">
+      <CombinationNew
+        color={[
+          'red',
+          'white',
+          'lightGray',
+          'gray',
+          'darkGray',
+          'green',
+          'pine',
+          'olive',
+          'blue',
+          'navy',
+          'midnight',
+          'purple',
+          'orchid',
+          'eggplant',
+          'maroon',
+          'watermelon',
+          'orange',
+          'transparent',
+          'transparentDarkGray',
+          'lightWash',
+          'darkWash',
+        ]}
+      >
+        {(props) => <Box width={60} height={60} rounding="circle" {...props} />}
+      </CombinationNew>
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      description={`The \`rounding\` property sets a border radius for the \`Box\`. Options are "circle" or "pill" for fully rounded corners or 0-8 representing the radius in 4px increments.`}
+      title="Rounding"
+    >
+      <CombinationNew rounding={['pill', 'circle', 0, 1, 2, 3, 4, 5, 6, 7, 8]}>
+        {(props) => (
+          <Box color="gray" width={props.rounding === 'pill' ? 120 : 70} height={70} {...props} />
+        )}
+      </CombinationNew>
+    </MainSection.Subsection>
+    <MainSection.Subsection description="" title="Opacity">
+      <CombinationNew opacity={[0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1]}>
+        {(props) => <Box color="darkGray" width={60} height={60} {...props} />}
+      </CombinationNew>
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      description={`
+    Box uses Flexbox to achieve layout. If you're new to flex layout, please read the excellent [CSS-Tricks guide to flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).
```

Comment:
Similar question here about if we should include flex stuff in the Box docs or point people to Flex instead